definitions/divorce/json/State/State.json

```diff
@@ -340,15 +340,6 @@
     "DisplayOrder": 22,
     "TitleDisplay": "${[CASE_REFERENCE]}: ${D8PetitionerLastName} vs ${D8RespondentLastName}"
   },
-  {
```

Comment:
Is this not needed on prod?

Reply:
The only thing that uses it is the "allowPetitionerToAmend" event.
This event is present in prod, but its authorisations are all in the non-prod files.
So I believe it isn't.